excretor/src/models.rs

```diff
@@ -3,18 +3,32 @@ use sqlx::prelude::FromRow;
 #[derive(FromRow)]
 pub struct Message {
     pub id: i32,
+    pub channel_name: String,
+    pub user_id: String,
     pub text: String,
-    pub user: User,
+    // Timestamp
+    pub ts: String,
+    // Thread timestamp
+    pub thread_ts: String,
+    // If it is a thread, id of the user who sent the parent message
+    pub parent_user_id: String,
```

Comment:
Since this can be NULL for threads, it should be an `Option<String>` instead

Reply:
I believe it will be empty, not null.